feat: chain autoswap (#139)
BoltzExchange/boltz-client: docs/autoswap.md

### Thresholds

Autoswap will create normal swaps when the outbound balance goes below the configured `outboundBalance` `outboundBalancePercent`.

> The same happens if the inbound balance goes below the configured `inboundBalance` or `inboundBalancePercent`.

Reviewer: ```suggestion
The same way, autoswap will create reverse swaps when the inbound balance goes below the configured `inboundBalance` or `inboundBalancePercent`.
```

If the percentage values are configured, the absolute thresholds will be calculated based on the channel capacity.

#### Single threshold

> If only one of the two thresholds is configured, the balance target will always be the full channel capacity.

Reviewer: ```suggestion
If `inboundBalance` is configured as threshold, the rebalancing target will always be the full channel capacity.
```

**Example**
- `inboundBalance` is set to 200k sats 
- Current inbound balance of our 500k sats channel is 100k. 

> - Result: A 400k sats reverse swap since the inbound balance is below the set threshold.

Reviewer: ```suggestion
- Result: A 400k sats reverse swap will be executed, to bring back inbound inbound balance to the full 500k sats channel capacity.```
